composer.json

```diff
@@ -68,46 +68,46 @@
         "sylius/resource": "^1.7",
         "sylius/resource-bundle": "^1.7",
         "sylius/theme-bundle": "^2.1.1",
-        "symfony/asset": "^4.4 || ^5.4",
-        "symfony/config": "^4.4 || ^5.4",
-        "symfony/console": "^4.4 || ^5.4",
-        "symfony/dependency-injection": "^4.4 || ^5.4",
-        "symfony/doctrine-bridge": "^4.4 || ^5.4",
-        "symfony/event-dispatcher": "^4.4 || ^5.4",
-        "symfony/expression-language": "^4.4 || ^5.4",
-        "symfony/filesystem": "^4.4 || ^5.4",
-        "symfony/finder": "^4.4 || ^5.4",
-        "symfony/form": "^4.4 || ^5.4",
-        "symfony/framework-bundle": "^4.4 || ^5.4",
-        "symfony/http-foundation": "^4.4 || ^5.4",
-        "symfony/http-kernel": "^4.4 || ^5.4",
-        "symfony/intl": "^4.4 || ^5.4",
-        "symfony/messenger": "^4.4 || ^5.4",
+        "symfony/asset": "^5.4",
+        "symfony/config": "^5.4",
+        "symfony/console": "^5.4",
+        "symfony/dependency-injection": "^5.4",
+        "symfony/doctrine-bridge": "^5.4",
+        "symfony/event-dispatcher": "^5.4",
+        "symfony/expression-language": "^5.4",
+        "symfony/filesystem": "^5.4",
+        "symfony/finder": "^5.4",
+        "symfony/form": "^5.4",
+        "symfony/framework-bundle": "^5.4",
+        "symfony/http-foundation": "^5.4",
+        "symfony/http-kernel": "^5.4",
+        "symfony/intl": "^5.4",
+        "symfony/messenger": "^5.4",
         "symfony/monolog-bundle": "^3.5",
-        "symfony/options-resolver": "^4.4 || ^5.4",
+        "symfony/options-resolver": "^5.4",
         "symfony/polyfill-iconv": "^1.17",
         "symfony/polyfill-intl-icu": "^1.22",
         "symfony/polyfill-mbstring": "^1.17",
         "symfony/polyfill-php80": "^1.17",
-        "symfony/process": "^4.4 || ^5.4",
-        "symfony/property-access": "^4.4 || ^5.4",
-        "symfony/property-info": "^4.4 || ^5.4",
-        "symfony/proxy-manager-bridge": "^4.4 || ^5.4",
-        "symfony/routing": "^4.4 || ^5.4",
-        "symfony/security-bundle": "^4.4 || ^5.4",
-        "symfony/security-core": "^4.4 || ^5.4",
-        "symfony/security-csrf": "^4.4 || ^5.4",
-        "symfony/security-guard": "^4.4 || ^5.4",
-        "symfony/security-http": "^4.4 || ^5.4",
-        "symfony/serializer": "^4.4 || ^5.4",
+        "symfony/process": "^5.4",
+        "symfony/property-access": "^5.4",
+        "symfony/property-info": "^5.4",
+        "symfony/proxy-manager-bridge": "^5.4",
+        "symfony/routing": "^5.4",
+        "symfony/security-bundle": "^5.4",
+        "symfony/security-core": "^5.4",
+        "symfony/security-csrf": "^5.4",
+        "symfony/security-guard": "^5.4",
+        "symfony/security-http": "^5.4",
+        "symfony/serializer": "^5.4",
         "symfony/string": "^5.3",
```

Comment:
Since this PR also _secretly_ drops support for Symfony 5.2 and 5.3, maybe also bump this requirement.
```suggestion
        "symfony/string": "^5.3",
```

Reply:
wrong regexp 👍